### DISCUSSION TITLE
articles/%5B%EB%B2%88%EC%97%AD%5D-indigo-la-end---%E7%9B%B2%E7%9B%AE%E3%81%A0%E3%81%A3%E3%81%9F-%EA%B0%80%EC%82%AC-%EB%B2%88%EC%97%AD

### DISCUSSION THREAD

**Discussion by giscus** [obvoso/obvoso-blog#32] [Comments]
# articles/%5B%EB%B2%88%EC%97%AD%5D-indigo-la-end---%E7%9B%B2%E7%9B%AE%E3%81%A0%E3%81%A3%E3%81%9F-%EA%B0%80%EC%82%AC-%EB%B2%88%EC%97%AD

오의역 다수

https://www.obvoso.site/articles/%5B%EB%B2%88%EC%97%AD%5D-indigo-la-end---%E7%9B%B2%E7%9B%AE%E3%81%A0%E3%81%A3%E3%81%9F-%EA%B0%80%EC%82%AC-%EB%B2%88%EC%97%AD

**Comment by Feedbaek** [comment]
와 JPOP!
